Lib/test/test_socket.py

```diff
@@ -2300,6 +2300,49 @@ def testCrucialConstants(self):
         socket.CAN_ISOTP
         socket.SOCK_DGRAM
 
+    @unittest.skipUnless(hasattr(socket, "SOL_CAN_ISOTP"),
+                         'Constants from isotp.h required for this test.')
```

Comment:
```suggestion
    @unittest.skipUnless(hasattr(socket, "SOL_CAN_ISOTP"),
    					 "missing <linux/can/isotp.h>")
```

Might seems silly but I think it's better to be concise when we can (also, it's not just any isotp.h header, but really `<linux/can/isotp.h>`).